plugins/Diagnostics/Diagnostic/AbstractPrivateDirectories.php

```diff
@@ -18,6 +19,7 @@ abstract class AbstractPrivateDirectories implements Diagnostic
 {
     protected $privatePaths = [];
     protected $accessiblePaths = []; // used like a set, but hashtable used underneath anyway, so map simpler php way
+    protected $addedDisableHttpDiagnosticsWarning = false;
```

Comment:
@geekdenz not too important but you shouldn't need this property anymore now that it is in the execute method.